documentation/SA1649.md

```diff
@@ -18,7 +18,7 @@
 ## Cause
 
 The file name of a C# code file does not match the first type declared in the file. For generics that are defined as
-`Class1<T>` the name of the file needs to be `Class1{T}.cs` or `Class1``1.cs` depending on the `fileNamingConvention`
+`Class1<T>` the name of the file needs to be `Class1{T}.cs` or `Class1```1.cs` depending on the `fileNamingConvention`
```

Comment:
Yeah, that work as well, and is probably even more readable when you edit. I've updated the PR to use this form.